src/PiWeb.Import.Sdk/Modules/ImportFormat/IImportFormat.cs

```diff
@@ -42,25 +36,31 @@ public interface IImportFormat
     #region methods
 
     /// <summary>
-    /// Decides what should be done with the given <paramref name="importGroup"/>.
-    /// See <see cref="ImportAction"/> for more information.
+    /// Gets the import format configuration. This configuration specifies which import format settings
+    /// are supported by this import format and which default values should be used for these settings.
     /// </summary>
-    /// <param name="importGroup">The potential import group containing the primary file to analyze.</param>
-    /// <param name="context">Provides context information such as other files in the import folder.</param>
-    ValueTask<ImportAction> DecideImportAction(IImportGroup importGroup, IGroupContext context);
-
+    /// <returns>The import format configuration.</returns>
+    IImportFormatConfiguration CreateConfiguration(ICreateImportFormatConfigurationContext context)
+    {
+        return new StandardImportFormatConfiguration();
+    }
+    
     /// <summary>
-    /// Creates data to be imported by parsing a given import group.
+    /// Creates an import group filter associated with this import format. The import group filter is responsible for
+    /// filtering and completing import groups consisting of import files that should be handled by this import format.  
     /// </summary>
-    /// <param name="importGroup">The import group to parse.</param>
-    /// <param name="context">Provides information about the import context.</param>
-    Task<ImportData> ParseImportData(IImportGroup importGroup, IParseContext context);
+    /// <param name="context">Provides context information.</param>
+    /// <returns>The created import group filter.</returns>
+    IImportGroupFilter CreateImportGroupFilter(ICreateImportGroupFilterContext context);
```

Comment:
Please add a comment what the plugin writer can expect to happen when that method throws an exception (difference between FileSourceException and other exceptions).